src/data_hub/plotting.py

```diff
@@ -0,0 +1,126 @@
+"""module to make plots from a Deep Origin dataframe"""
+
+from beartype import beartype
+from bokeh.io import show
+from bokeh.layouts import column, row
+from bokeh.models import ColumnDataSource, CustomJS, Select
+from bokeh.plotting import figure
+from deeporigin.data_hub.dataframe import DataFrame
+
+
+@beartype
+def create_interactive_scatter(df: DataFrame):
```

Comment:
bokeh code is fairly inscrutable but i hope these comments help

Reply:
They do.